lib/experian/precise_id/client.rb

```diff
@@ -18,9 +18,23 @@ def send_answers(options = {})
 
       def submit_request(request)
         raw_response = super
-        log_raw_response(raw_response)
         response = Response.new(raw_response.body)
+        check_response_for_errors(response, raw_response)
         [request,response]
+      rescue => e
+        log_error(e)
+        log_raw_response(raw_response)
+      end
+
+      def check_response_for_errors(response, raw_response)
+        if Experian.logger && response.error? && (response.error_code.nil? && response.error_message.nil?)
```

Comment:
If `error_code` is something and `error_message` is `nil` it would not log the raw response. Is that the intended behavior?